reference/docs/conversion/sakai_11_mysql_conversion.sql

```diff
@@ -681,27 +681,27 @@ alter table lti_tools add pl_importitem tinyint(4) default 0;
 alter table lti_tools add fa_icon varchar(1024);
 alter table lti_tools add tool_proxy_binding mediumtext;
 
-ALTER TABLE lti_content MODIFY (     title VARCHAR(1024) );
-ALTER TABLE lti_content MODIFY (     pagetitle VARCHAR(1024) );
-ALTER TABLE lti_content MODIFY (     consumerkey VARCHAR(1024) );
-ALTER TABLE lti_content MODIFY (     secret VARCHAR(1024) );
-alter table lti_content modify ( 	custom varchar(65536) );
-
-ALTER TABLE lti_tools MODIFY (     title VARCHAR(1024) );
-ALTER TABLE lti_tools MODIFY (     pagetitle VARCHAR(1024) );
-ALTER TABLE lti_tools MODIFY (     consumerkey VARCHAR(1024) );
-ALTER TABLE lti_tools MODIFY (     secret VARCHAR(1024) );
-alter table lti_tools modify (	custom varchar(65536) );
-
-ALTER TABLE lti_deploy MODIFY (     title VARCHAR(1024) );
-ALTER TABLE lti_deploy MODIFY (     pagetitle VARCHAR(1024) );
-ALTER TABLE lti_deploy ADD (     allowcontentitem tinyint(4) DEFAULT 0 );
-ALTER TABLE lti_deploy MODIFY (     reg_key VARCHAR(1024) );
-ALTER TABLE lti_deploy MODIFY (     reg_password VARCHAR(1024) );
-ALTER TABLE lti_deploy ADD (	reg_ack CLOB );
-ALTER TABLE lti_deploy MODIFY (     consumerkey VARCHAR(1024) );
-ALTER TABLE lti_deploy MODIFY (     secret VARCHAR(1024) );
-ALTER TABLE lti_deploy ADD (     new_secret VARCHAR(1024) );
+ALTER TABLE lti_content MODIFY title VARCHAR(1024);
+ALTER TABLE lti_content MODIFY pagetitle VARCHAR(1024);
+ALTER TABLE lti_content MODIFY consumerkey VARCHAR(1024);
+ALTER TABLE lti_content MODIFY secret VARCHAR(1024);
+alter table lti_content modify custom varchar(65536);
+
+ALTER TABLE lti_tools MODIFY title VARCHAR(1024);
+ALTER TABLE lti_tools MODIFY pagetitle VARCHAR(1024);
+ALTER TABLE lti_tools MODIFY consumerkey VARCHAR(1024);
+ALTER TABLE lti_tools MODIFY secret VARCHAR(1024);
+alter table lti_tools modify custom varchar(65536);
+
+ALTER TABLE lti_deploy MODIFY title VARCHAR(1024);
+ALTER TABLE lti_deploy MODIFY pagetitle VARCHAR(1024);
+ALTER TABLE lti_deploy ADD allowcontentitem tinyint(4) DEFAULT 0;
+ALTER TABLE lti_deploy MODIFY reg_key VARCHAR(1024);
+ALTER TABLE lti_deploy MODIFY reg_password VARCHAR(1024);
+ALTER TABLE lti_deploy ADD reg_ack TEXT;
+ALTER TABLE lti_deploy MODIFY consumerkey VARCHAR(1024);
+ALTER TABLE lti_deploy MODIFY secret VARCHAR(1024);
+ALTER TABLE lti_deploy ADD new_secret VARCHAR(1024);
 
 CREATE TABLE lti_memberships_jobbbs (
```

Comment:
On our case sensitive deployment all the table names are lower case and the `org.sakaiproject.lti.impl.DBLTIService#init` looks to create them all as lower case. 

I think MySQL is not case sensitive for columns names (http://dev.mysql.com/doc/refman/5.7/en/identifier-case-sensitivity.html)

Personally I like all the SQL grammar in upper case and schema references in lower case as I find it easier to read, but that's just a personal thing.